src/graph/utils/pm/nested_matcher.cpp

```diff
@@ -252,7 +255,7 @@ bool match_node_inputs(const binding_t &b, match_context_t *ctx,
     if (node_inputs_matcher.get_node()->get_inputs().size()
             == VARIADIC_INPUT_NUM) {
         matching_status = node_inputs_matcher.match_variadic_inputs();
-    } else if (!has_commutative_inputs(node_inputs_matcher.get_op())) {
+    } else if (!is_commutative_input(node_inputs_matcher.get_op())) {
```

Comment:
what will happen if this function is called without a set of input ports?

Reply:
It will return directly `commutative_inputs_enabled_`